default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
```

Comment:
It does not make much sense to me to have `default.nix` that is not instantiable.